src/rabbit_epmd_monitor.erl

```diff
@@ -27,6 +27,8 @@
 
 -define(SERVER, ?MODULE).
 -define(CHECK_FREQUENCY, 60000).
+-define(LOCAL_IPV6_ADDRESS, {0,0,0,0,0,0,0,1}).
```

Comment:
I believe `LOOPBACK` would be more accurate than `LOCAL` in the name, because any IP address assigned to this host is local.